src/compiler.ts

```diff
@@ -4630,13 +4642,21 @@ export class Compiler extends DiagnosticEmitter {
               expr = module.if(leftExpr, module.i32(1), rightExpr);
             }
           }
+          let inheritCondi =
+            condKind == ConditionKind.TRUE
+              ? ConditionKind.FALSE
+              : condKind == ConditionKind.FALSE
+              ? ConditionKind.TRUE
+              : ConditionKind.UNKNOWN;
```

Comment:
It will be great to create util something like:
```ts
function invertedCondition(kind: ConditionKind): ConditionKind {
  if (kind == ConditionKind.TRUE) return ConditionKind.FALSE;
  if (kind == ConditionKind.FALSE) return ConditionKind.TRUE;
  return kind;
}
```
and later:
```ts
let inheritCondi = invertedCondition(condKind);
```

Reply:
thanks!